content/ko/integrations/amazon_elasticache.md

```diff
@@ -8,6 +8,7 @@ categories:
 - cloud
 - configuration & deployment
 - log collection
+custom_kind: 통합
```

Comment:
should this frontmatter key be translated?

Reply:
short answer is that it probably shouldn't be, but we have work arounds so this doesn't affect rendering.   for example the tile should still be here https://docs-staging.datadoghq.com/guacbot/translation-pipeline/ko/integrations/?q=amazon_elasticache and the page itself works fine https://docs-staging.datadoghq.com/guacbot/translation-pipeline/ko/integrations/amazon_elasticache/

Reply:
Thanks! And sounds good, I checked a handful of the pages and things were still looking correct